bindings/nodejs/build.rs

```diff
@@ -1,19 +1,5 @@
-// Copyright 2022 Datafuse Labs
```

Comment:
**request change**: OpenDAL is going to be an ASF project. It's required that all files shouled be licensed under Apache 2.0

Reply:
Yes. I know this. Will update them.